server/database/queries/addExercise.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../config/connection');
+
+const addExercise = (userID, exerciseID, exerciseTime) => {
```

Comment:
Just for consistency, I’d suggest renaming “exerciseTime” to “exerciseDuration”, since that’s the name used in the controller (and is similar to the table column name). That makes the code a little less confusing to others, and they can search the codebase for that name and find results in both files.